psi4/driver/driver_findif.py

```diff
@@ -280,11 +293,41 @@ def _initialize_findif(mol: Union["qcdb.Molecule", core.Molecule],
     # Get settings for CdSalcList, then get the CdSalcList.
     method_allowed_irreps = 0x1 if mode == "1_0" else 0xFF
     # core.get_option returns an int, but CdSalcList expect a bool, so re-cast
-    salc_list = core.CdSalcList(mol, method_allowed_irreps, t_project, r_project)
+    if core.get_option("FINDIF", "molsym"):
+        try:
+            import molsym
+        except ImportError:
+            print("Download & install MolSym at: https://github.com/NASymmetry/MolSym")
```

Comment:
Try following this pattern: https://github.com/psi4/psi4/blob/master/psi4/driver/driver.py#L809-L810 . Can use GH for now since pypi and conda aren't ready.